tools/parity_scale_decoder/samples/trigger.json

```diff
@@ -6,7 +6,9 @@
                 {
                     "Mint": {
                         "object": {
-                            "U32": 1
+                            "Numeric": {
+                                "U32": 1
+                            }
```

Comment:
This is problematic. Please make sure that this serialises and deserialises the way it used to.